phylogenetic/defaults/config.yaml

```diff
@@ -53,10 +53,15 @@ subsample:
     focal:
       query: division == 'WA'
       subsample_max_sequences: 300
-    contextual:
+    contextual_similar:
       query: country == 'USA' & division != 'WA'
       proximity_target: focal
-      subsample_max_sequences: 50
+      subsample_max_sequences: 25
+    contextual_other:
+      query: country == 'USA' & division != 'WA'
+      subsample_max_sequences: 25
+      exclude_samples:
+        - contextual_other
```

Comment:
I'm confused by this `exclude_samples` block, it's excluding itself?

Reply:
Sorry, that was unfinished copy/paste. Fixed:

https://github.com/nextstrain/WNV/blob/b5d03c64019e6b75a4f09fe3191c0ad1b97f851e/phylogenetic/defaults/config.yaml#L63-L64